### PR TITLE
Egfad 148

### PR DESCRIPTION
**Pull request by duncanmeech**
Import to empty block to create a new block it working.
Basic parsing of DNA is working.
E2E tests for all functionality working.
Import command is available from main menu and block context menu.
NOT implemented are a few of the UX stories concerning rules about the selected block(s). I will need Max add some API support for that first. Card has been added for Max

### CODE REVIEW

File: src/styles/importdnaform.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.importdnaform {
+  width: 500px;
+
+  textarea {
+    font-size: 1.25rem;
+    font-family:monospace;
+    border-radius: var(--radius);
```
Comment (florencio-mazzoldi) [inline src/styles/importdnaform.css]: From the console when you run the server: WARNING in ./~/css-loader!./~/postcss-loader!./src/styles/importdnaform.css
postcss-custom-properties: /Users/flo/Dev/genome-designer/src/styles/importdnaform.css:7:5: variable '--radius' is undefined and used without a fallback

WARNING in ./~/css-loader!./~/postcss-loader!./src/styles/importdnaform.css
postcss-custom-properties: /Users/flo/Dev/genome-designer/src/styles/importdnaform.css:9:5: variable '--inputcolor' is undefined and used without a fallback

**Comment by florencio-mazzoldi** [comment]
Test failure:
[Register Then Signin] Test Suite
=================================

Running:  Test homepage registration, signout and sign in.
 ✔ Expected homepage element to be present
 ✔ Expected form to become visible
 ✔ Passed [equal]: Sign In == Sign In
 ✔ Expected form to become visible
 ✔ Testing if element .error.visible occurs on the page 6 times.
 ✔ expected to be signed in
 ✔ expected user to be signed in
 ✔ expected menu to appear
 ✔ expected to be signed out
 ✔ expected a sign out button
 ✔ Expected sign in dialog to become visible
 ✖ Testing if element .error.visible occurs on the page 1 times.  - expected "1" but got: 0
    at signout (/Users/flo/Dev/genome-designer/test-e2e/fixtures/signin.js:10:13)
    at Object.module.exports.Test homepage registration, signout and sign in. (/Users/flo/Dev/genome-designer/test-e2e/tests/register-then-signin.js:13:5)
    at Module.call (/Users/flo/Dev/genome-designer/node_modules/nightwatch/lib/runner/module.js:63:34)
    at /Users/flo/Dev/genome-designer/node_modules/nightwatch/lib/runner/testcase.js:97:29
    at _fulfilled (/Users/flo/Dev/genome-designer/node_modules/nightwatch/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/flo/Dev/genome-designer/node_modules/nightwatch/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/flo/Dev/genome-designer/node_modules/nightwatch/node_modules/q/q.js:796:13)
    at /Users/flo/Dev/genome-designer/node_modules/nightwatch/node_modules/q/q.js:556:49
    at runSingle (/Users/flo/Dev/genome-designer/node_modules/nightwatch/node_modules/q/q.js:137:13)
    at flush (/Users/flo/Dev/genome-designer/node_modules/nightwatch/node_modules/q/q.js:125:13)

FAILED:  1 assertions failed and 11 passed (4.712s)

**Comment by florencio-mazzoldi** [comment]
Can you please also uncomment the .only in the test/server/data/querying.spec.js line 16, and see if you can get the tests to pass?

File: src/styles/importdnaform.css (hunk 1 above)
Comment (duncanmeech) [inline src/styles/importdnaform.css]: Fixed by importing correct variables file

**Pull request merged**